packages/pn-personagiuridica-webapp/src/pages/Deleghe.page.tsx

```diff
@@ -1,37 +1,45 @@
 import { useEffect, useState } from 'react';
```

Comment:
I should remove all te code useless for the purpose of the issue

Reply:
done